### PR TITLE
Fix presskit

### LINKED ISSUE #8827: Add FAQ to Press Kit
Based on the volume of interview requests that Habitica receives, we've decided to implement a FAQ section on the Press Kit. All of the text can be found here: https://docs.google.com/document/d/17Ys40wZOkP4kSUDHE3aQN8Ayaxz5Wpkryg3lcbn5BUc/edit?usp=sharing

This should appear towards the bottom of the press kit page, below the images.

The "Marketing/Social Media Inquiries" section of the contact page should also be changed to link to the Press Kit page, since it will have my email at the bottom.  (I notice that the Report a Bug link also needs to be changed from GH to the Report A Bug Guild.)

Let me know if there are any questions!

### LINKED ISSUE #9143: PressKit page is broken
The presskit page on the new website is broken, see https://habitica.com/static/press-kit.

There are some issues in the code that prevent it from working

Code like this

```
img.img-rendering-auto.press-img(:src="`assets/images/presskit/${category}/${img}.png`")
```

won't work because for webpack urls to work they cannot use computed properties. So either define them manually or move everything to `/website/static` and then link the images as `/static/${pathToImageInStaticFolder}`

Also there are some files in `client/assets/images/presskit` that aren't images, in particular a `.zip` and a `.mp4`. Those should go into `website/static`

### PR DESCRIPTION
**Pull request by librarianmage**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Your_Local_Install_to_Modify_Habitica%27s_Website_and_API for more info)

[//]: # (Put Issue # or URL here, if applicable. This will automatically close the issue if your PR is merged in)
Fixes #9143, Resolves #8827

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)

[//]: # (Put User ID in here - found on the Habitica website at User Icon > Settings > API)

----
UUID:

### CODE REVIEW

**Comment by librarianmage** [comment]
screenshot 2017-11-06 at 7 23 20 pm
screenshot 2017-11-06 at 7 23 39 pm

**Comment by librarianmage** [comment]
The website pages are out of date again; should new ones be created?

**Comment by lemoness** [comment]
Yes, definitely! That's actually a good task for @beffymarroo, when she has
some time.

**Comment by librarianmage** [comment]
@lemoness Would this PR still be able to be merged without the updated images?

**Comment by lemoness** [comment]
We hope to have these new screenshots ready this week, so let's hold off on
merging until those are ready. Thanks for your patience!

**Comment by librarianmage** [comment]
@lemoness How are the screenshots going?

**Comment by beffymaroo** [comment]
Files are here: 
https://drive.google.com/file/d/1yKVRQEbDS_HwYHU1S2nnPbEHOmqEJXCs/view?usp=sharing

Should be accessible with the link, but let me know if there's a permissions adjustment I missed.

My apologies I am not able to attach directly, the file is too large, apparently.

**Comment by librarianmage** [comment]
@beffymaroo I can see it

**Comment by librarianmage** [comment]
@beffymaroo Are any of the images from the previous presskit still wanted?

**Comment by beffymaroo** [comment]
@MathWhiz I think any of the old images that were still wanted are already in the updated folder- we just replaced outdated ones.

**Comment by librarianmage** [comment]
Updated!

File: website/client/components/static/pressKit.vue [hunk 1]
```diff
@@ -1,24 +1,35 @@
...
+        h2(v-b-toggle="QA.question", tabindex="0", role="button", v-html="$t(QA.question)")
+        b-collapse(:id="QA.question", accordion="pkAccordian", role="tabpanel")
+          p(v-html="$t(QA.answer)")
+    
+    p {{ $t('pkMoreQuestions') }}
 </template>
 
 <script>
-  // @TODO: How to use images?
+  import bCollapse from 'bootstrap-vue/lib/components/collapse';
```
Comment (TheHollidayInn) [inline website/client/components/static/pressKit.vue]: We have updated the way we use bootstrap Vue. Can you remove these two imports

File: website/client/components/static/pressKit.vue [hunk 2]
```diff
@@ -28,50 +39,96 @@
...
+            answer: 'pkAnswer7',
+          },
+          {
+            question: 'pkQuestion8',
+            answer: 'pkAnswer8',
+          },
+        ],
       };
     },
+    components: {
```
Comment (TheHollidayInn) [inline website/client/components/static/pressKit.vue]: Can you also remove the components and directives objects.

**Review by TheHollidayInn — COMMENTED** [review]
Sorry for the delay! Looks good. Can you update the bootstrap vue imports then we are gtg.

**Comment by librarianmage** [comment]
@TheHollidayInn Fixed!

**Comment by librarianmage** [comment]
Are these no longer added to my entry in the Hall?

**Comment by SabreCat** [comment]
@MathWhiz: Normally we note PRs as a way of keeping track of when the next tier should be awarded, but you're already maxed out on tiers thanks to all your hard work!

**Comment by librarianmage** [comment]
oh, right! I hadn't considered that, derp :p

**Pull request merged**